lang_python.adoc

```diff
@@ -13,130 +13,155 @@ To generate Python modules, Kaitai Struct compiler should be invoked with
 ----
 +
-and then run `pip install -r requirements.txt` to update all your
-dependencies.
+.
 
 * Otherwise, you can just install the package manually using *pip*
   (Python package manager):
 +
 [source,shell]
-pip install --upgrade --pre git+https://github.com/kaitai-io/kaitai_struct_python_runtime.git
+python3 -m pip install --upgrade --pre git+https://github.com/kaitai-io/kaitai_struct_python_runtime.git
+
 
-[[add-dependencies]]
-=== Adding dependencies
+[[install-dependencies]]
+=== Installing dependencies for legacy Pythons manually
 
-<<ksy_reference#enums,Enums>> implementation in Python requires a enum
-support as described in
+<<ksy_reference#enums,Enums>> implementation in Python requires
+a enum support as described in
 https://www.python.org/dev/peps/pep-0435/[PEP-0435]. This support is
 available out of the box in standard libraries since Python v3.4, or
-since v2.4 using https://pypi.org/project/enum34/[enum34] PyPi
-compatibility layer.
+since v2.4 using https://pypi.org/project/enum34/[enum34] polyfill.
 
-* On Debian / Ubuntu, this library can be installed with `sudo apt-get
-  install python-enum34`
-* Otherwise, one can use just `sudo pip install enum34`
+* One usually doesn't need to install this lib manually, `pip` will install it
+   automatically if it is needed.
+* One can also install it manually:
+** One can use just `python -m pip install enum34`. Don't do it on
+     later Pythons or the apps using just `enum` will be broken!
+** On Debian / Ubuntu, this library can be installed with
+    `sudo apt-get install -y python-enum34`.
 
-[[automation]]
+[[build-automation]]
 === Automation of compilation process
 
 https://setuptools.readthedocs.io/en/latest/[Setuptools] is the de-facto
-standard way of building and installing python packages. This library
+standard way of building and installing Python packages. This library
 allows extension with
 https://setuptools.readthedocs.io/en/latest/setuptools.html#extending-and-reusing-setuptools[plugins].
-https://github.com/KOLANICH have developed
-https://github.com/kaitaiStructCompile/kaitaiStructCompile.py[an extension to
-setuptools], allowing you to automate compilation of `*.ksy` files as the
-part of python module build process. The library provides a mostly
-declarative syntax to facilitate installation: you add a parameter
-`kaitai` to your `setup.py` where you enumerate files that you want
-compiled, output directory, compilation flags, and source code
-post-compilation transformations. Here are some examples:
-https://github.com/KOLANICH/NTMDTRead/blob/master/setup.py[1],
-https://github.com/KOLANICH/SpecprParser.py/blob/master/setup.py[2].
+https://gitlab.com/KOLANICH has developed
+https://gitlab.com/kaitaiStructCompile.py/kaitaiStructCompile.setuptools[an extension to setuptools] ( https://github.com/kaitaiStructCompile/kaitaiStructCompile.setuptools[GitHub mirror is also present]), allowing you to automate compilation of `*.ksy` files as the
```

Comment:
An additional space after the opening bracket here:

```suggestion
https://gitlab.com/kaitaiStructCompile.py/kaitaiStructCompile.setuptools[an extension to setuptools] (https://github.com/kaitaiStructCompile/kaitaiStructCompile.setuptools[GitHub mirror is also present]), allowing you to automate compilation of `*.ksy` files as the
```

You could also split this extra long line into several shorter ones to make the source more readable:

```suggestion
https://gitlab.com/kaitaiStructCompile.py/kaitaiStructCompile.setuptools[an extension to setuptools]
(https://github.com/kaitaiStructCompile/kaitaiStructCompile.setuptools[GitHub mirror is also present]),
allowing you to automate compilation of `*.ksy` files as the
```

Reply:
Done.